CONTRIBUTING.md

```diff
@@ -125,6 +126,32 @@ See [the API reference](https://docs.llamaindex.ai/en/latest/api_reference/index
 
 ---
 
+
+
+
+### Difference between AutoMerging, SentenceWindow and others, with respect to indexing, query engines and node parsers.
+
+Let's break down the differences between AutoMerging, SentenceWindow, and generic node parsers like sentence splitters:
+
+1. **AutoMerging**:
+   - **Functionality**: AutoMerging is primarily concerned with identifying and merging similar or related documents or data entities to reduce redundancy and improve efficiency. It involves algorithms that analyze the content of documents or data sets and automatically group together items that share common characteristics.
```

Comment:
Uhhh this isn't true -- it merges based on relationship

For example, a chunk is split into 4 sub chunks. If 2/4 sub chunks are retrieved, they are just replaced with the parent chunk